go.sum

```diff
@@ -1,5 +1,186 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+cloud.google.com/go v0.38.0/go.mod h1:990N+gfupTy94rShfmMCWGDn0LpTmnzTp2qbd1dvSRU=
```

Comment:
Most of these added lines are not needed. Can you try running `go mod tidy` at the root level of the repository and see if it removes lines from `go.sum` again?

Reply:
I am running `go mod tidy` at the root level (i.e. where the README.md, go.mod, etc... files live) and there is no changes made. My go version is: `go version go1.20.3 linux/amd64`.

Reply:
I'm not sure if this is an issue with Go or with the library. A go.sum file ideally only contains a single entry per dependency as only of them will be used (and checked). I will try to find out why that happens as I'd like to avoid adding so much noise.

Reply:
I don't understand this either. I did check, the large number of `go.sum` difference occur after I run the command `go get github.com/containrrr/shoutrrr@v0.7.1` or `go get -u github.com/containrrr/shoutrrr`

Reply:
Seems like it is expected, although I don't fully understand what it's good for. It's noisy, but it won't be included in the build, so I guess it's good to go like this.